src/nbl/video/utilities/ImageRegionIterator.cpp

```diff
@@ -130,10 +130,10 @@ size_t ImageRegionIterator::getMemoryNeededForRemainingRegions() const
     {
         const asset::IImage::SBufferCopy & region = regions[i];
 
-        auto imageExtentInBlocks = dstImageTexelBlockInfo.convertTexelsToBlocks(core::vector3du32_SIMD(region.imageExtent.width, region.imageExtent.height, region.imageExtent.depth));
+        auto imageExtentInBlocks = dstImageTexelBlockInfo.convertTexelsToBlocks(hlsl::uint32_t3(region.imageExtent.width, region.imageExtent.height, region.imageExtent.depth));
 
```

Comment:
@Erfan-Ahmadi its your code, review changes to this file

Things to look out for: `vectorSIMD` was implicitly 4D (unused coords 0) whereas Przemek is now using an appropriate sized thing

Reply:
I had used vector3du32_SIMD explicitly, uint32_t3 should be fine.
not much to review, I give it green light,